### PR TITLE
ISSUE-740 Clarify order of commands in README

### PR DESCRIPTION
**Pull request by lewismc**
This PR addresses #740

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,34 +45,29 @@ See public/licenses.txt
...
-* (Optional) For automatic spatial and temporal extraction from the search text, clone and set up an [EDSC-NLP](https://git.earthdata.nasa.gov/projects/EDSC/repos/edsc-nlp/browse) server
+* (Optional) For automatic spatial and temporal entity extraction from the search text, clone and set up an [EDSC-NLP](https://git.earthdata.nasa.gov/projects/EDSC/repos/edsc-nlp/browse) server
 
-### Initial setup
-
-Run
+### Application Configuration
 
-    bin/setup
+Use the templates at `config/application.yml.example` and `config/database.yml.example` to create `config/application.yml` and `config/database.yml` respectively. Update values as necessary. Note that the Earthdata Login (URS) Configuration is optional however without the Earthdata Login configuration, Earthdata Search's functionality will be limited. If you would like to set up Earthdata Login login, you will need to perform the following steps:
```
Comment (mreese84) [inline README.md]: A couple of grammar updates...
1) "Update values as necessary. Note that the Earthdata Login (URS) Configuration is optional. However, without the Earthdata Login configuration..."
2) "If you would like to set up Earthdata Login login, you will need to perform the following steps:" I don't think the second "login" is needed. Should read "If you would like to set up Earthdata Login, you will need..."

File: README.md [hunk 2]
```diff
@@ -45,34 +45,29 @@ See public/licenses.txt
...
 
-Note: This command will take a long time to run
-
-### Application configuration
+Register an account on [the Earthdata Login home page](https://urs.earthdata.nasa.gov/home).
```
Comment (mreese84) [inline README.md]: I think that the steps outlined to set up Earthdata Login should be a list (bullet or numbered, doesn't really matter to me.) But a list would read more clearly IMO.

File: README.md [hunk 3]
```diff
@@ -45,34 +45,29 @@ See public/licenses.txt
...
 
-Review `config/application.yml` and update values as necessary
+Create an application in the Earthdata Login console. Its callback URL should be `http://<domain>/urs_callback`.  Standard Rails development would be `http://localhost:3000/urs_callback`.
 
-#### (Optional) Earthdata Login (URS) Configuration
+Click the "Feedback" icon on the Earthdata Login page and request that your new application be placed in the ECHO application group (required for ECHO/CMR to recognize your tokens).
 
-Without the Earthdata Login Configuration, Earthdata Search's functionality will be limited. If you would like to set up Earthdata Login login, you will need to perform the following steps:
+Your Earthdata Login application's client ID will need to be saved in `config/application.yml`.
```
Comment (mreese84) [inline README.md]: Toward the end of the list, I might would add in something like "More information on Earthdata Login can be found here." and link to https://developer.earthdata.nasa.gov/urs/welcome

**Comment by lewismc** [comment]
@mreese84 thanks for review

**Comment by mreese84** [comment]
Closing this one up. We have completely redone our readme in our serverless migration. This will be OBE fairly soon.

**Pull request closed without merging**